q

```diff
@@ -0,0 +1,45 @@
+[1mdiff --git a/spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubMessageChannelBinderEmulatorIntegrationTests.java b/spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubMessageChannelBinderEmulatorIntegrationTests.java[m
```

Comment:
looks like some temp diff file sneaked in here.

Reply:
oops. Can't even blame the cat; this was me.